content/sensu-go/5.16/installation/install-sensu.md

```diff
@@ -220,8 +220,8 @@ To start using sensuctl, run `sensuctl configure` and log in with your user cred
 
 {{< highlight "shell" >}}
 sensuctl configure -n \
---username 'admin' \
---password 'P@ssw0rd!' \
+--username 'YOUR_USERNAME' \
+--password 'YOUR_PASSWORD' \
```

Comment:
This works, but I think we also need to update docker installation commands around line 101 (docker) and 108 (docker compose) to include init step.

Reply:
@cwjohnston I'm not totally sure how to do that -- Should there be separate sections for Docker in the Initialize section (like at line 91)?
Or would they need to run sensu-backend init again for every Docker container?

Reply:
@rgeniesse pointed out that current plans call for keeping Docker as-is (with default admin/P@ssw0rd!) for 5.16 and rolling out `sensu-backend init` for Docker in 5.17. See https://sensu.slack.com/archives/C60EEQFH8/p1576190537067300 for context.

Reply:
Yeah, so based on that we won't need to init the docker container; it'll be done for the users in the building of the image. So for those users, they won't notice a difference. But then in 5.17, they'll need to do it. At least, that's the plan.